app/page.tsx

```diff
@@ -4,7 +4,7 @@ import Slick from "@/components/commons/Slick";
 import Youtube from "@/components/index/Youtube";
 
 export default function Home(): JSX.Element {
-  const { images } = useStore();
+  const { images, youtubeMeta } = useStore();
```

Comment:
```suggestion
  const { images } = useStore();
```

사용안하는 데이터가 들어갔어용 !ㅠㅠ

Reply:
깃 컨플릭트 나면서 생긴거 같습니다. 최신 버전엔 없을 예정입니다!